content/healthchecks/readinessprobe.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Configure Readiness Probe"
+chapter: true
+weight: 10
+---
+
+# Configure Readiness Probe
+
+```
```

Comment:
Instead of downloading readiness-deployment.yaml on the intro, let's show the code here and have the user save the file as ~/environment/healthchecks/readiness-deployment.yaml. Explain/highlight the readinessProbe settings and what we expect to happen.